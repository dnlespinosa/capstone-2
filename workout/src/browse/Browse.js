import React, { useEffect, useState } from 'react';
import WorkoutApi from '../api/api';
import ProgramCard from '../Program/ProgramCard'
import axios from 'axios';

const Browse = ({ programs }) => {



    return (
        <>
            <div>
                <h1 style={{color: 'white'}}>BROWSE WORKOUTS</h1>
                {programs ? 
                
                    <div>
                        {programs.workouts.map(p => (
                            <ProgramCard 
                                key={p.name}
                                id={p.workout_id}
                                name={p.name}
                                exercises ={p.exercise_names} />

                        ))}
                    </div>
                 : <p>Sorry not results found</p>}
            </div>
        </>
    )
}

export default Browse