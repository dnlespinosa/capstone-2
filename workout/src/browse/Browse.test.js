import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import Browse from "./Browse";
import { UserProvider } from "../testUtils";


it("renders without crashing", function () {
    render(
        <MemoryRouter>
          <UserProvider>
            <Browse />
          </UserProvider>
        </MemoryRouter>,
    );
  });