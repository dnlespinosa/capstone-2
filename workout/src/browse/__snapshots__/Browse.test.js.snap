// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1
      style="color: white;"
    >
      BROWSE WORKOUTS
    </h1>
    <p>
      Sorry not results found
    </p>
  </div>
</DocumentFragment>
`;
